storybook/src/utils/AspectRatio/AspectRatio.tsx

```diff
@@ -4,12 +4,7 @@
  */
 
 import { AspectRatioProps } from '@amsterdam/design-system-react/src/common/types'
-import { HTMLAttributes, PropsWithChildren } from 'react'
+import { HTMLAttributes } from 'react'
 
-/** Renders examples in Storybook. Not for reuse. */
-export const AspectRatio = ({
-  aspectRatio,
-  children,
-}: AspectRatioProps & PropsWithChildren<HTMLAttributes<HTMLSpanElement>>) => (
-  <span className={`ams-aspect-ratio--${aspectRatio}`}>{children}</span>
-)
+/** Mock component to render examples in Storybook. Not for reuse. */
```

Comment:
If you update this comment, please do so for all utility classes (margin, gap, visually hidden).